fe/fe-core/src/main/java/com/starrocks/connector/parser/trino/Trino2SRFunctionCallTransformer.java

```diff
@@ -109,10 +108,6 @@ private static void registerArrayFunctionTransformer() {
                         new PlaceholderExpr(1, Expr.class), new PlaceholderExpr(2, Expr.class)
                 )))));
 
-        // 2. concat -> array_concat
```

Comment:
could you add more description for this PR, why previous method won't work?

Reply:
Done